projects/plugins/crm/tests/php/bootstrap.php

```diff
@@ -79,12 +79,22 @@
  * Load Jetpack CRM.
  */
 function _jpcrm_manually_load_plugin() {
+	// Load the main plugin file
 	require_once JETPACK_CRM_TESTS_ROOT . '/../../ZeroBSCRM.php';
 
-	// Run all register_activation_hook() functions.
+	// For tests, we need to manually initialize the plugin
 	global $zbs;
-	$zbs->install();
-	zeroBSCRM_notifyme_createDBtable();
+	if ( class_exists( 'zeroBSCRM' ) && ! isset( $zbs ) ) {
+		$zbs = zeroBSCRM::instance();
+	}
+
+	// Run installation if needed
+	if ( isset( $zbs ) && is_object( $zbs ) && method_exists( $zbs, 'install' ) ) {
+		$zbs->install();
+		if ( function_exists( 'zeroBSCRM_notifyme_createDBtable' ) ) {
```

Comment:
Same thing here...if `$zbs` is instantiated, that means it's run the `__construct()`, which means it's run `$zbs->includes()`, which means this function does exist. There's no way to get to this point and not have the function exist.